apps/rollups/graphql/mock/server.ts

```diff
@@ -1,16 +1,17 @@
+import { loadFilesSync } from '@graphql-tools/load-files';
+import { mergeTypeDefs } from '@graphql-tools/merge';
 import { addMocksToSchema } from '@graphql-tools/mock';
 import { makeExecutableSchema } from '@graphql-tools/schema';
-import { readFileSync } from 'fs';
 import { createYoga } from 'graphql-yoga';
 import { createServer } from 'node:http';
 import { join } from 'path';
-import { BigInt, Int, Float, String } from './custom-scalars';
-
-const typeDefs = readFileSync(join(process.cwd(), 'graphql/schema.graphql'), {
-    encoding: 'utf-8',
+import { BigInt, Float, Int, String } from './custom-scalars';
+const typesArray = loadFilesSync(join(__dirname, '../schema'), {
+    recursive: true,
 });
-
+const typeDefs = mergeTypeDefs(typesArray);
```

Comment:
@dandheedge, 
I think a rollback to the old way to load schema would be better defaulting to the latest i.e. v0.9. Because the merge here probably has problems with this merge action. They are in a sense different "universes" instead of a universe broken apart for maintenance sake that needs to be reassembled to be loaded in this mock server. I would just update the README session if necessary highlighting that other schema versions are available to be used on developer needs.